src/App.tsx

```diff
@@ -12,6 +12,28 @@ export class App extends Component<{}, State> {
     movies: [],
   };
 
+  findMovieInArray = (movieToCheck: Movie) => {
```

Comment:
Since returned value of this function is Boolean, name it like this
```suggestion
  checkIsMovieInArray = (movieToCheck: Movie) => {
```

Reply:
Done